brakeman.json

```diff
@@ -0,0 +1,103 @@
+{
+  "scan_info": {
+    "app_path": "/Users/rachalcassity/va/platform-console",
```

Comment:
I don't think you mean to commit brakeman.json and rubocop.json. Could you add them to your global gitignore?